src/Symfony/Component/Messenger/Bridge/Doctrine/Transport/Connection.php

```diff
@@ -236,25 +223,10 @@ public function reject(string $id): bool
     public function setup(): void
     {
         $configuration = $this->driverConnection->getConfiguration();
-        // Since Doctrine 2.9 the getFilterSchemaAssetsExpression is deprecated
-        $hasFilterCallback = method_exists($configuration, 'getSchemaAssetsFilter');
-
-        if ($hasFilterCallback) {
-            $assetFilter = $this->driverConnection->getConfiguration()->getSchemaAssetsFilter();
-            $this->driverConnection->getConfiguration()->setSchemaAssetsFilter(null);
-        } else {
-            $assetFilter = $this->driverConnection->getConfiguration()->getFilterSchemaAssetsExpression();
-            $this->driverConnection->getConfiguration()->setFilterSchemaAssetsExpression(null);
-        }
-
+        $assetFilter = $configuration->getFilterSchemaAssetsExpression();
```

Comment:
isn't that the deprecated method we don't want to use according to the comment above
> Since Doctrine 2.9 the getFilterSchemaAssetsExpression is deprecated

Reply:
fixed